src/toasts/toasts.stories.html

```diff
@@ -0,0 +1,21 @@
+<article class="stories-spacing">
+  <h2>OpenMRS Spacing, Padding, and Margin</h2>
+  The OpenMRS styleguide provides css classes for padding and margin.
+  <section>
+    <h3>Padding</h3>
+
+    <div class="classname-columns">
```

Comment:
@Andu033 could you also add a tab on how to use the showToast in JSX ? check the latest commit on how to achieve this

Reply:
I think i figured a way but the onclick doesn't work. But if i change to other story and come back the toast pop up.